docs/WINDOWS_SPECS.md

```diff
@@ -0,0 +1,455 @@
+# Especificaciones Técnicas - Windows (WinAPI)
+
+## 🪟 Implementación Windows
+
+La implementación para Windows utiliza WinAPI nativo para crear ventanas overlay semi-transparentes con capacidades de layered window y transparencia por canal alfa.
+
+## 📦 Dependencias
+
+### Dependencias del Sistema
+- Windows 10/11 (recomendado)
+- Windows 8.1+ (mínimo)
+- Visual Studio Build Tools 2019+ o Visual Studio Community
+- Windows SDK 10.0+
+
+### Dependencias de Rust
```

Comment:
This is not needed. It is better to comment `Cargo.toml` or use `[target.*.dependencies]`
```suggestion
```